src/graphql/mutation/shopAddOnePrintKivaCard.graphql

```diff
@@ -0,0 +1,5 @@
+mutation {
+	shop {
+		addOnePrintKivaCard
+	}
```

Comment:
It feels like you're missing some in this file. If you look at the updateKivaCardAmount.graphql file, or the updateDonation.graphql files, we're passing through more information which I sense is needed for this to work properly.